scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsColumnsFactory.java

```diff
@@ -87,6 +112,15 @@ protected void buildCommonRecordAttributes(Job item, Record record) {
         }
 
         record.setAttribute(DURATION_ATTR.getName(), duration);
+
+        for (GridColumns extraColumn : EXTRA_COLUMNS) {
+            String columnName = extraColumn.getName();
+            if (columnName.startsWith(GENERIC_INFORMATION_MATCHER)) {
```

Comment:
The usage would be simpler to understand if you extract the retrieval/construction of the key/columnName in a dedicated utility class.